### PR TITLE
Add `update_version` to the default fields for `plugin` and `theme` commands

### PR DESCRIPTION
**Pull request by sejas**
- Related:
  - https://github.com/wp-cli/wp-cli/issues/5859
  - https://github.com/wp-cli/extension-command/issues/198

## Description

I'm adding the field `update_version` as a default field for `plugin` and `theme` commands. If no `update_version` is available we return an empty string following the same pattern as `version` itself.

Now `wp plugin list` and `wp theme list` display a table similar to:
```
+------------+----------+--------+---------+----------------+-------------+
| name       | status   | update | version | update_version | auto_update |
+------------+----------+--------+---------+----------------+-------------+
| akismet    | inactive | none   | 5.3     |                | off         |
| hello      | inactive | none   | 1.7.2   |                | off         |
| Zombieland | inactive | none   | 0.1.0   |                | off         |
| no-mail    | must-use | none   |         |                | off         |
| polyfills  | must-use | none   |         |                | off         |
+------------+----------+--------+---------+----------------+-------------+
```

```
+-------------------+----------+--------+---------+----------------+-------------+
| name              | status   | update | version | update_version | auto_update |
+-------------------+----------+--------+---------+----------------+-------------+
| twentytwentyfour  | active   | none   | 1.0     |                | off         |
| twentytwentythree | inactive | none   | 1.3     |                | off         |
| twentytwentytwo   | inactive | none   | 1.6     |                | off         |
+-------------------+----------+--------+---------+----------------+-------------+
```

## Testing instructions

1. Run `composer behat -- features/plugin-update.feature`
2. Run `composer behat -- features/plugin.feature`
3. Run `composer behat -- features/upgradables.feature`
4. Run `composer behat -- features/theme.feature`
5. Observe the tests pass

### CODE REVIEW

File: features/plugin.feature [hunk 1]
```diff
@@ -131,8 +131,8 @@ Feature: Manage WordPress plugins
 
     When I run `wp plugin list`
     Then STDOUT should be a table containing rows:
-      | name               | status   | update    | version |
-      | wordpress-importer | active   | available | 0.5     |
+      | name               | status   | update    | version | update_version   |
+      | wordpress-importer | active | available | 0.5     | {UPDATE_VERSION} |
```
Comment (schlessera) [inline features/plugin.feature:135]: Alignment needs a fix here.

File: features/theme.feature [hunk 2]
```diff
@@ -97,8 +101,8 @@ Feature: Manage WordPress themes
 
     When I run `wp theme list`
     Then STDOUT should be a table containing rows:
-      | name  | status   | update    | version   |
-      | p2    | active   | available | 1.4.1     |
+      | name  | status   | update    | version   | update_version |
+      | p2    | inactive | available | 1.4.1     | {UPDATE_VERSION} |
```
Comment (schlessera) [inline features/theme.feature:105]: Alignment needs to be fixed here.

File: features/theme.feature [hunk 3]
```diff
@@ -77,10 +77,14 @@ Feature: Manage WordPress themes
...
+    When I run `wp theme list --name=p2 --field=update_version`
+    Then STDOUT should not be empty
+    And save STDOUT as {UPDATE_VERSION}
+
     When I run `wp theme list`
     Then STDOUT should be a table containing rows:
-      | name  | status   | update    | version   |
-      | p2    | inactive | available | 1.4.2     |
+      | name  | status   | update    | version   | update_version   |
+      | p2    | active | available | 1.4.2     | {UPDATE_VERSION} |
```
Comment (schlessera) [inline features/theme.feature:87]: Alignment needs a fix here.

**Review by schlessera — CHANGES_REQUESTED** [review]

**Review by swissspidy — COMMENTED** [review]
Besides the alignment details this looks good 👍

**Comment by sejas** [comment]
Thanks for the quick review! I fixed the alignment issues. I also updated some comments on the functions.

And I realized that the `json` format is returning null instead of an empty string. Is that something I need to change?
I would appreciate where that part of the code lives.

` wp plugin list --format=json`

```
[
    {"name":"akismet","status":"active","update":"none","version":"5.3","update_version":null}
    ,{"name":"hello","status":"inactive","update":"none","version":"1.7.2","update_version":null}
    ,{"name":"Zombieland","status":"inactive","update":"none","version":"0.1.0","update_version":null}
    ,{"name":"no-mail","status":"must-use","update":"none","version":"","update_version":null}
    ,{"name":"polyfills","status":"must-use","update":"none","version":"","update_version":null}
]
````

The table, and csv format are working correctly returning an empty string.

I'll also solve the merging conflicts.

**Comment by sejas** [comment]
> And I realized that the json format is returning null instead of an empty string. Is that something I need to change?
I would appreciate where that part of the code lives.

I found it:  https://github.com/wp-cli/extension-command/pull/380/commits/9749bd2a293c12609ef4df95741453438e9029d1

**Review by swissspidy — APPROVED** [review]

**Pull request merged**